src/Private/System/Web/HttpUtility.cs

```diff
@@ -57,7 +57,7 @@ public override string ToString()
                     return "";
                 }
 
-                StringBuilder sb = new StringBuilder();
+                StringBuilder sb = new();
```

Comment:
Nitpick: In some other places, you're using `var`. That still avoids the type name repetition, but it also keeps the type name and parameter list closer to each other, which seems more easily maintainable to me. For your consideration. :)

Reply:
I agree with you. I'll search for similar issues in the codebase, merge to master and release a new version tonight. A huge "thank you" for all the time and effort you've invested in this. If you encounter any other issues, don't be a stranger and let me know.